src/registry/FileSystemRegistry.ts

```diff
@@ -106,6 +107,11 @@ export class FileSystemRegistry extends SynchronousRegistry implements IRegistry
     this.createFile({ filePath: addressesPath, data: toYamlString(addresses) });
   }
 
+  addWarpRouteConfig(warpConfig: WarpRouteDeployConfig, fileName: string): void {
+    const filePath = path.join(this.uri, this.getWarpRoutesConfigPath(), fileName);
```

Comment:
Can you write the config right next to the existing warpCoreConfig path?

Reply:
I.e. as per https://www.notion.so/hyperlanexyz/Warp-Speed-Part-1-Warp-Config-in-Registry-17b6d35200d680e8b465df2bd32cf42b?pvs=4#17b6d35200d6803da2ebf79b9e04af50

Reply:
fixed

Reply:
https://github.com/hyperlane-xyz/hyperlane-registry/pull/516/files